contributing/chestertons-fence.md

```diff
@@ -0,0 +1,79 @@
+# Chesterton's Fence
+
+Hello :wave:
+You're probably here becomes someone directed you to this post in response to you suggesting an idea about how Exercism could be improved.
```

Comment:
```suggestion
You're probably here because someone directed you to this post in response to you suggesting an idea about how Exercism could be improved.
```